backend/pkg/backend/backend.go

```diff
@@ -220,9 +221,33 @@ func convertSpecDiffToEventDiff(diff *_spec.APIDiff) (originalRet, modifiedRet [
 	return originalRet, modifiedRet, nil
 }
 
+func getPathItemForVersion(v3PathItem *spec.PathItem, version _spec.OASVersion) interface{} {
+	if v3PathItem == nil {
+		return v3PathItem
+	}
+
+	switch version {
+	case _spec.OASv2:
+		log.Errorf("Converting to OASv2 path item")
+		v2PathItem, err := openapi2conv.FromV3PathItem(&spec.T{Components: spec.Components{}}, v3PathItem)
+		if err != nil {
+			log.Errorf("failed to convert v3 path item to v2, keeping v3: %v", err)
```

Comment:
Failed to convert.
Don't we want to return an error in this case (the function can't do what the user requested)?

Reply:
would it not be better to show diff in v3 format rather than not showing anything?

Reply:
we do, but it's a bit misleading. maybe better to call it getPathItemForVersionOrOriginal?